src/handlers/shared/stop.ts

```diff
@@ -0,0 +1,56 @@
+import { Context } from "../../types";
+import { addCommentToIssue, closePullRequestForAnIssue } from "../../utils/issue";
+
+export async function stop(context: Context, issue: Context["payload"]["issue"], sender: Context["payload"]["sender"], repo: Context["payload"]["repository"]) {
+  const { logger } = context;
+  const issueNumber = issue.number;
+  const out = { output: null };
+
+  // is it an issue?
+  if (!issue) {
+    logger.info(`Skipping '/stop' because of no issue instance`);
+    return out;
+  }
```

Comment:
```suggestion
```

We can probably get rid of these checks. I know they were necessary in the old bot because of the poor type management.